test/unit/scheduleConfig.js

```diff
@@ -0,0 +1,38 @@
+const chai = require('chai');
+const config = require('../../app/lib/scheduleConfig');
+
+const expect = chai.expect;
+
+const disableScheduler = process.env.DISABLE_SCHEDULER;
+const etlSchedule = process.env.ETL_SCHEDULE;
+
+function resetEnv() {
+  process.env.disableScheduler = disableScheduler;
```

Comment:
Should this be `process.env.DISABLE_SCHEDULER = disableScheduler`?
Looks like i've missed it in the other PRs too 😞